Source/Libraries/CorruptCore/Objects.cs

```diff
@@ -1723,6 +1663,52 @@ public List<BlastUnit> GetRasterizedUnits(string vmdToRasterize = null)
             return l.Layer;
         }
 
+        private bool SetDomainAndAddress(bool useSourceDomain)
```

Comment:
Could you rename this to something slightly more descriptive? Maybe SetRealDomainAndAddress or something. The idea behind the code here is it converts a VMD to the underlying domains, but it has this recursive breakdown element to it where if the VMD is non-contiguous, it needs to recursively break the unit down. Might make more sense to do a bigger refactor at some point to make that code flow more clear.

Reply:
Changed to `SetRealDomainAndAddress`.

Thanks for the context. Sounds like this code may be worth revisiting in the future.